src/core/components/ChatInput.tsx

```diff
@@ -12,13 +12,21 @@ import { IdbImage } from './IdbImage';
 import ButtonIcon from './ButtonIcon';
 import { useTheme } from '../../shared/theme/useTheme';
 import { ChatCompletionContentPart } from 'openai/resources/index';
+import { PDFiumLibrary } from '@hyzyla/pdfium/browser/base64';
```

Comment:
note:  using the basr64 library is not recommended for prod.  https://pdfium.js.org/docs/init-library#browser